scrum-board-system/src/components/userstories/AddUserStoryModal.js

```diff
@@ -0,0 +1,216 @@
+      <div class="input-group-prepend">
+        <div class="input-group-text">
+          <input type="checkbox" aria-label="Checkbox for following text input" checked={taskState?.done} name={`tasks[${index}].done`} ref={register}/>
+        </div>
+      </div>
+      <input type="text" class="form-control" aria-label="Text input with checkbox" defaultValue={taskState?.description} name={`tasks[${index}].description`} ref={register} onKeyDown={handleKey}/>
+    </div>
+  )
+}
+const TasksContainer = ({tasks,register}) => {
+  const [tasksMutated,setTaskMutated]=useState(!!tasks?tasks:[{}])
+  const newTaskEnter=()=>{
+    setTaskMutated([...tasksMutated,{}])
+  }
+    return (
+      <div class="form-group row">
+        <div class="col-sm-2">Tasks</div>
+        {tasksMutated && tasksMutated.map((task,index) => (
+          <>
+          <div className="col-sm-2"></div>
+          <Task key={task.id}
+          task={task} register={register} index={index} onEnterPressed={newTaskEnter}/>
+          </>
+        ))}
+        
+      </div>
+    )
+}
+
+const UserStory = (props) => {
+  const { sprints,userStoryEdit, onSubmit,setLabelTagsHandler } = props
+  const labelsCatalog=props.labels
+  const {title,description,weight, labels, tasks, sprint, id } = userStoryEdit
+  const { handleSubmit, register, error,control} = useForm()
+  const { fields, append, prepend, remove, swap, move, insert } = useFieldArray({
+    control,
+    name: "tasks"
+  }); 
+
+
+  return (
+    <>
+      <form>
+      <FormControl type="text" readOnly hidden ref={register} name="id" id="id" value={ id } />
+        <div class="form-group row">
+          <label for="title" class="col-sm-2 col-form-label">Title</label>
+          <div class="col-sm-10">
+            <input type="text" class="form-control" ref={register} name="title" id="title" defaultValue={title}/>
+          </div>
+        </div>
+        <div class="form-group row">
+          <label for="description" class="col-sm-2 col-form-label">Description</label>
+          <div class="col-sm-10">
+            <input type="textarea" class="form-control" ref={register} name="description" id="description" defaultValue={description} />
+          </div>
+        </div>
+        <div class="form-group row">
+        <label for="weight" class="col-sm-2 col-form-label">Weight</label>
+        <div class="col-sm-10">
+          <input type="text" class="form-control" ref={register} name="weight" id="weight" defaultValue={weight}/>
+        </div>
+      </div>
+      <Sprint sprint={sprint} sprints={sprints} register={register} />
+      <LabelContainer labels={labels} labelsCatalog={labelsCatalog} setLabelTagsHandler={setLabelTagsHandler} />
+      <TasksContainer tasks={tasks} register={register}/>
+      <input type="button" onClick={handleSubmit(onSubmit)} className="btn btn-primary" value={(id!=null)?"SAVE":"ADD"}/>
+    </form>
+        </>
+  )
+}
```

Comment:
line space here, linter could fix automatically

Reply:
Could you explain me this on a call?